geps/overview.md

```diff
@@ -65,18 +65,53 @@ against them. At this stage, the GEP should be targeting the "Implementable"
 stage.
 
 ### 4. Implement the GEP as "Experimental"
+
+
 With the GEP marked as "Implementable", it is time to actually make those
 proposed changes in our API. In some cases, these changes will be documentation
 only, but in most cases, some API changes will also be required. It is important
 that every new feature of the API is marked as "Experimental" when it is
 introduced. Within the API, we use `<gateway:experimental>` tags to denote
 experimental fields.
 
-Before these changes are released, they will also need to be documented.
-GEPs that have not been both implemented and documented before a release
-cut off will be excluded from the release.
+Some other requirements must be met before marking a GEP `Experimental`:
+- the graduation criteria to reach `Standard` MUST be filled out
```

Comment:
new line needed for list formatting: https://deploy-preview-2097--kubernetes-sigs-gateway-api.netlify.app/geps/overview/

```suggestion
Some other requirements must be met before marking a GEP `Experimental`:

- the graduation criteria to reach `Standard` MUST be filled out
```